File: Strings/__1000__/1243/1243.java
import java.util.Scanner;

public class Main {
    
    public static void main(String[] args){
        
        Scanner in = new Scanner(System.in);
        
        while(in.hasNextLine()){
            String[] frase = in.nextLine().split(" ");
            int media = media_frase(frase);
            
            System.out.println(
                media <= 3 ? 250 : (media > 3 && media <= 5 ? 500 : 1000)
            );
        }
        
        in.close();
    }
    
    public static int media_frase(String[] frase) {
        int totalPalavras = 0;
        int somatoriaPalavras = 0;
    
        for (String palavra : frase) {
            if (palavra.endsWith(".")) {
                palavra = palavra.substring(0, palavra.length() - 1);
            }
    
            if (palavra.matches("[A-Za-z]+")) {
                somatoriaPalavras += palavra.length();
                totalPalavras++;
            }
        }
    
        if (totalPalavras == 0) return 0;
        return somatoriaPalavras / totalPalavras;
    }
}

File: Strings/__1000__/1243/1243.ts
const datas = require("fs").readFileSync("/dev/stdin", "utf8");
const inputs = datas.split("\n");
inputs.pop()

const media_frase = (frase) => {
    let total_palavras = 0;
    let somatoria_palavras = 0;
    
    for (const palavra of frase) {
        const palavra_limpa = palavra.endsWith(".") ? palavra.slice(0, -1) : palavra;
        
        if (/^[A-Za-z]+$/.test(palavra_limpa)) {
            somatoria_palavras += palavra_limpa.length;
            total_palavras++;
        }
    }
    
    if (total_palavras === 0) return 0;
    return Math.floor(somatoria_palavras / total_palavras);
};

for (const line of inputs) {
    const media = media_frase(line.split(" "));
    console.log(media <= 3 ? 250 : (media <= 5 ? 500 : 1000));
}